<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Finder</class>
 <widget class="QWidget" name="Finder">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>688</width>
    <height>683</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="lineEdit_path">
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>Путь к папке</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="file_review">
       <property name="text">
        <string>Обзор</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLineEdit" name="lineEdit_pattern">
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>Паттерн</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_filter">
       <item>
        <property name="text">
         <string>Строки</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Бинарный</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Байты</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBox_file_size">
     <property name="text">
      <string>Размер файлов</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Искать в файлах размером меньше</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_file_size_filter">
       <property name="styleSheet">
        <string notr="true">background-color: gray;</string>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>байт</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="find">
     <property name="text">
      <string>Искать</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListWidget" name="listWidget_pathes"/>
   </item>
   <item>
    <widget class="QPushButton" name="open_path_file">
     <property name="text">
      <string>Открыть папук с файлом</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
